components/imageCard.js

```diff
@@ -1,11 +1,19 @@
 import {useState, useEffect} from "react";
-import Image from "next/image";
+import Image from "next/legacy/image";
 import styled from "styled-components";
+import deleteIcon from "../public/deleteIcon.svg";
+import deleteIconFilled from "../public/deleteIconFilled.svg";
+import likeIcon from "../public/likeIcon.svg";
+import likeIconFilled from "../public/likeIconFilled.svg";
+import {useContext} from "react";
+import {ImageContext} from "./ImageContext";
 
 export default function ImageCard() {
+  // const {setFavorite} = useContext(ImageContext);
   const [dataArray, setDataArray] = useState([]);
   const [isLoading, setLoading] = useState(false);
-  const [displayImage, setDisplayImage] = useState(false);
```

Comment:
shouldn't `isLoading` be initialized with `true`? Otherwise I guess you would see "No data" instead of "Loading..." during load time.

Reply:
One could think but if the page is refreshed that "Loading..."-message can be seen until the image shows up. I changed it to "true" and it still does the same as before.